shacl12-overview/index.html

```diff
@@ -373,61 +373,211 @@ <h2>Introduction</h2>
-			</p>
+      <p>
+        SHACL 1.2 caters for <a href="https://www.w3.org/TR/rdf12-new/">new developments in RDF 1.2 and SPARQL 1.2</a>, formalises versions of original SHACL documents published only as Notes and takes SHACL into new areas such as rules for inferencing and profile defining.
+      </p>
+      <p>
+        The original SHACL specifications, published in 2015 and which we now call SHACL 1.1, are listed in the <a href="#shacl-1.1">SHACL 1.1</a> section below. SHACL 1.2's specifications are also listed below in the <a href="#shacl-1.2">SHACL 1.2</a> section.
+      </p>
+      <p>
+        following are the major new things in SHACL 1.2 and the specifications they appear in.
+      </p>
+      <table id="new-items" class="term-table">
+        <tbody>
+          <tr>
+            <th>
+              New Feature
+            </th>
+            <th>
+              Description
+            </th>
+            <th>
+              SHACL 1.2 Specification
+            </th>
+          </tr>
+          <tr>
+            <td>
+              Derived Properties
+            </td>
+            <td>
+              SHACL 1.1 shapes and constraints could only operate on asserted triples in a graph and inferencing was left as an optional pre-processing step, using languages like RDF Schema and OWL. SHACL 1.2 introduces its own inferencing and reasoning capabilities, making SHACL more self-contained, and covering different use cases than RDFS/OWL.
+            </td>
+            <td>
+              <a href="https://www.w3.org/TR/shacl12-core/">Core</a>,<br />
+              <a href="https://www.w3.org/TR/shacl12-node-expr/">Node Expr</a>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              Flexible Target Nodes
+            </td>
+            <td>
+              In SHACL 1.2, node expressions can be used in multiple places beside derived properties. In particular, they can also be used to compute the target nodes of a shape. This provides greater flexibility than the built-in target types such as <code>sh:targetClass</code> and <code>sh:targetSubjectsOf</code>.
+            </td>
+            <td>
+              <a href="https://www.w3.org/TR/shacl12-node-expr/">Node Expr</a>
+            </td>
+          </tr>
+          <tr>
+            <td>Inference Rules</td>
+            <td>
+              <p>
+                The new <em>Inferencing Rules</em> specification allows the representation and execution of <a href="https://en.wikipedia.org/wiki/Datalog">Datalog</a>-like inference rules which have a straight-forward mapping to SPARQL <code>CONSTRUCT</code> statements for execution, and integrate with other SHACL features such as node expressions.
```

Comment:
It seems odd to title this side-spec `Inferencing Rules` when the things it discusses are generally referred to as `Inference Rules`. I suggest we change most if not all instances of the former, including the title of the other document, to the latter.
```suggestion
            <td>Inference Rules</td>
            <td>
              <p>
                The new <em>Inference Rules</em> specification allows the representation and execution of <a href="https://en.wikipedia.org/wiki/Datalog">Datalog</a>-like inference rules which have a straight-forward mapping to SPARQL <code>CONSTRUCT</code> statements for execution, and integrate with other SHACL features such as node expressions.
```